### PR TITLE
[REF] pos_self_order: move demo data from pos_self_order_iot

### PR DESCRIPTION
**Pull request by loouis-t**
We added a demo Kiosk to the PoS in the `pos_self_order_iot` module. We moved it to the `pos_self_order` module, as it can benefit both.

Enterprise PR: [https://github.com/odoo/enterprise/pull/69000](https://github.com/odoo/enterprise/pull/69000)

### CODE REVIEW

File: addons/pos_self_order/models/pos_config.py [hunk 1]
```diff
@@ -321,17 +321,50 @@ def action_open_wizard(self):
...
+            'payment_method_ids': not_cash_payment_methods_ids,
+            'limit_categories': True,
+            'iface_available_categ_ids': restaurant_categories,
+            'iface_splitbill': True,
+            'module_pos_restaurant': True,
+        })
+        config.write({'self_ordering_mode': 'kiosk'})
+        import logging
+        _logger = logging.getLogger(__name__)
+        _logger.critical(config.self_ordering_mode)
```
Comment (Ysoroko) [inline addons/pos_self_order/models/pos_config.py:365]: saw it

**Review by davidmonnom — APPROVED** [review]
LGTM @odoo/rd-security can you override CI for this PR ? :) Thx

**Review by xmo-odoo — COMMENTED** [review]
@robodoo override=ci/security

**Comment by davidmonnom** [comment]
@robodoo r+

**Pull request closed without merging**